learn/get-started/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Get Started
+---
+
+The best way to start your journey with Stellar is to create a [test network](./test-net.html) account and send your first transaction.
+
+In this tutorial we will be using our [JavaScript SDK](https://www.stellar.org/developers/js-stellar-sdk/learn/index.html) but you can use other [SDKs](https://www.stellar.org/developers/horizon/learn/#libraries) as well.
+
+## Installing `stellar-sdk`
+
+First create your NodeJS project if `package.json` file is not present in your working directory:
+```
+npm init
+```
+
+Then install our JavaScript SDK:
+```
+npm install --save stellar-sdk
+```
+
+## Creating a test network account
+
+First you have to generate your key pair.
+
+`generate_keypair.js` file:
+```js
+var StellarSdk = require('stellar-sdk');
+var keypair = StellarSdk.Keypair.random();
+
+console.log('Account ID:');
+console.log(keypair.accountId());
+console.log('Secret Seed:');
```

Comment:
Fixed in https://github.com/stellar/docs/commit/890632e7bbb5c52511a76a8ca70351e9c1459e68